commit 7ba50fed3e670700e263bcbb47385f7727f9e21f (refs/changes/75/75/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-07-14T04:25:41+00:00 (3 years, 3 months ago)
    
    joyent/node-forkexec#4 NPM package should not ship development files

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..db5dad8
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,4 @@
+examples/
+tools/
+test/
+Makefile*
